Doc/library/typing.rst

```diff
@@ -23,10 +23,9 @@
 
 --------------
 
-This module provides runtime support for type hints. The most fundamental
-support consists of the types :data:`Any`, :data:`Union`, :data:`Callable`,
-:class:`TypeVar`, and :class:`Generic`. For a specification, please see
-:pep:`484`. For a simplified introduction to type hints, see :pep:`483`.
+This module provides runtime support for type hints. For the original
+specification of the typing system, see :pep:`484`. For a simplified
+introduction to type hints, see :pep:`483`.
```

Comment:
`Union` is "soft-deprecated"; `Callable` is deprecated; following PEP-695, we prefer for people not to instantiate `TypeVar`s directly or to inherit from `Generic` directly. I feel like this list doesn't serve much purpose anymore, so it's better to just delete it.

Also, describe PEP-484 as the "original specification" rather than "the specification". The typing system has evolved a lot since then.